lib/runner/variant-reassignment.js

```diff
@@ -5,14 +5,13 @@ import TransactionService from '../services/transaction-manager'
 
 export default class VariantReassignment {
 
-  constructor (client, logger, options = {}, blackList = [], retainExistingAttributes = []) {
+  constructor (client, logger, options = {}, retainExistingData = []) {
     this.unfinishedTransactions = []
     this.firstRun = true
     this.customObjectService = null // build custom object service
-    this.blackList = blackList
+    this.retainExistingData = retainExistingData
```

Comment:
Do you have any plan how to deal with `price` and `assets` fields? Will they be in `retainExistingData` by default?

Reply:
Why do you want to have default attributes? Let the user make the choice?

Reply:
Oki.. we should add `retainExistingData` to the CLI parameters in product import then, right?

Reply:
Yep, we will add it to product import as a param.